core/src/main/java/com/amplitude/core/platform/Timeline.kt

```diff
@@ -42,6 +42,10 @@ open class Timeline {
     fun remove(plugin: Plugin) {
         // remove all plugins with this name in every category
         plugins.forEach { (_, list) ->
+            val wasRemoved = list.remove(plugin)
```

Comment:
why call remove twice?

Reply:
Good catch!

Reply:
In Kotlin, removing an element from a list that does not exist, nothing dramatic happens and no exception. Nice catch! 

Removed the second `remove()` at https://github.com/amplitude/Amplitude-Kotlin/pull/171/commits/6db25e4c2a92b5e3ce31f920aa950ebb44451073.